viz-lib/src/visualizations/table/columns/number.tsx

```diff
@@ -37,7 +37,7 @@ export default function initNumberColumn(column: any) {
 
   function prepareData(row: any) {
     return {
-      text: format(row[column.name]),
```

Comment:
Please remove these unnecessary submissions, which may cause confusion to other developers.